karabo_data/geometry2.py

```diff
@@ -1050,6 +1129,15 @@ def to_distortion_array(self):
         """
         return super().to_distortion_array()  # Overridden only for docstring
 
+    @classmethod
+    def _tile_dims(cls, tileno):
+        # Which part of the array is this tile?
+        tile_offset = tileno * cls.frag_ss_pixels
+        ss_dims = tile_offset, tile_offset + cls.frag_ss_pixels - 1
+        fs_dims = 0, cls.frag_fs_pixels - 1 # Every tile covers the full pixel range
+        return ss_dims, fs_dims
```

Comment:
This doesn't look right for LPD - there's a 2 x 8 tile layout, so it doesn't cover the full range in either dimension. Here's the `_tile_slice` method in #142:

https://github.com/European-XFEL/karabo_data/blob/f14ebb4f6e6ad35bc6c3740bbe6ba9d7cd11ad3b/karabo_data/geometry2.py#L1142-L1153

The tile layout within the module is in the top-right of this diagram:

![lpd_layout](https://user-images.githubusercontent.com/327925/59290466-89584500-8c70-11e9-9adc-a91e70acfe4c.png)

Reply:
Ok I'll fix this